js/websocket.js

```diff
@@ -0,0 +1,14 @@
+if (window.location.hostname.indexOf("lndo") > 0) {
+  var HOST = 'ws://muk_ws.lndo.site';
+}
+else {
+  var HOST = location.origin.replace(/^http/, 'ws');
```

Comment:
What happens here on 'https' ?

Reply:
I am NOT sure! This is copied from the Heroku websocket demo. I believe it will be fine because there is a `wss` and `ws`, so that last `s` is optional and will translated to secure websockets, depending on the deployment environment.